arm/Microsoft.Network/bastionHosts/deploy.bicep

```diff
@@ -55,7 +55,7 @@ param azureBastionpLogsToEnable array = [
   'BastionAuditLogs'
 ]
 
-var azureBastionDiagnosticsLogs = [for log in azureBastionpLogsToEnable: {
+var diagnosticsLogs = [for log in azureBastionpLogsToEnable: {
```

Comment:
Rename `azureBastionpLogsToEnable` too. I wonder why I added it back them. Usually I only did that if there was a conflict (e.g. 2 different sets of logs)

Reply:
Done!